tiny_gltf.h

```diff
@@ -4949,7 +4949,7 @@ static bool ParseMesh(Mesh *mesh, Model *model, std::string *err, const json &o,
     }
   }
 
-  if (mesh->primitives.size() == 0) {
+  if (mesh->primitives.size() == 0 && required_properties) {
```

Comment:
Im assuming this variable is a boolean?

Reply:
yes https://github.com/ambi-robotics/tinygltf/pull/4/files#diff-83ec95ad61ca1d30a028693fedc246df7e98d4ae8caa8821c87b339201be9545R4930